<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Account::Internal::DistanceRulesWidget</class>
 <widget class="QWidget" name="Account::Internal::DistanceRulesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>10</y>
     <width>401</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="distanceRulesLabel">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Types</string>
      </property>
      <property name="buddy">
       <cstring>distancerulesComboBox</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QComboBox" name="distanceRulesComboBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>200</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteButton">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="typeLabel">
      <property name="minimumSize">
       <size>
        <width>45</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Rule</string>
      </property>
      <property name="buddy">
       <cstring>typeEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="2" colspan="2">
     <widget class="QLineEdit" name="typeEdit">
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QDoubleSpinBox" name="valueDoubleSpinBox"/>
    </item>
    <item row="3" column="3">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="valueLabel">
      <property name="text">
       <string>Value</string>
      </property>
      <property name="buddy">
       <cstring>valueDoubleSpinBox</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="uidLabel">
      <property name="text">
       <string>Uid</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="2">
     <widget class="QLabel" name="distanceRulesUidLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="4">
     <widget class="QTextBrowser" name="textBrowser">
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;You can create a rule, and a value for this rule.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;For example, &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;distance in this area,&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;value : 0,6 &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;it's mean 0,6 euro (or local currency) / kilometer.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
